docs/install.rst

```diff
@@ -0,0 +1,202 @@
+************
+Installation
+************
+
+.. _astropy-main-req:
+
+Requirements
+============
+
+``SkyPy`` has the following strict requirements:
+
+- `Python <https://www.python.org/>`_ |minimum_python_version| or later
+
+- `Numpy`_ |minimum_numpy_version| or later
+
+- `scipy`_ |minimum_scipy_version| or later: To power a variety of features
+  in several modules.
+
+- `astropy`_ |minimum_astropy_version| or later
+
+- `networkx <https://networkx.github.io/>`_
+
+- `PyYAML <https://pyyaml.org>`_
+
+``SkyPy`` also depends on other packages for optional features:
+
+- `matplotlib <https://matplotlib.org/>`_ 2.0 or later: To provide plotting
+  functionality that `astropy.visualization` enhances.
+
+- `pytest-astropy <https://github.com/astropy/pytest-astropy>`_
+
+- `sphinx-astropy <https://github.com/astropy/sphinx-astropy>`_
+
+- `camb <<https://github.com/astropy/sphinx-astropy>`_
+.. - `setuptools <https://setuptools.readthedocs.io>`_: Used for discovery of
+  entry points which are used to insert fitters into `astropy.modeling.fitting`.
+
+
+The following packages can optionally be used when testing:
+
+- `tox <https://tox.readthedocs.io/en/latest/>`_: Used to automate testing
+  and documentation builds.
```

Comment:
It might be better to describe how to install the requirements from the setup.cfg, including the various extras?